### PR TITLE
Fixes threshold bug

### LINKED ISSUE #1010: threshold throws error if used more than once consecutively
### 
Screenshot 2019-04-13 at 15 24 36

if we use the threshold module more than once consecutively, it throws an error.

### Please show us where to look

https://beta.sequencer.publiclab.org

### What's your PublicLab.org username?

> This can help us diagnose the issue:

### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by Divy123**
Fixes #1010  

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Divy123** [comment]
@jywarren @HarshKhandeparkar please review!!

File: examples/lib/defaultHtmlSequencerUi.js [hunk 1]
```diff
@@ -47,6 +47,8 @@ function DefaultHtmlSequencerUi(_sequencer, options) {
   
     if(!newStepName) newStepName = arguments[0]
 
+    if(typeof newStepName !== "string")
+    newStepName = $(".item")[0].getAttribute("data-value");
```
Comment (HarshKhandeparkar) [inline examples/lib/defaultHtmlSequencerUi.js]: you can use jQuery `.attr()`. Btw, what is `.item`?

Comment (Divy123) [reply examples/lib/defaultHtmlSequencerUi.js]: Actually this is used because of selectize. Nevertheless, our code is working perfectly.

Comment (Divy123) [reply examples/lib/defaultHtmlSequencerUi.js]: @HarshKhandeparkar is it ok to you?

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlSequencerUi.js]: What exactly is `.item` in selectize?

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlSequencerUi.js]: Also, you can use `$.attr()` method.

Comment (Divy123) [reply examples/lib/defaultHtmlSequencerUi.js]: Its the class given to a div generated by selectize after  consuming the select tag.

Comment (Divy123) [reply examples/lib/defaultHtmlSequencerUi.js]: I can but that does not make any difference.

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlSequencerUi.js]: It shortens the code, makes it look better, makes it consistent

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlSequencerUi.js]:
```suggestion
    newStepName = $(".item").attr("data-value");
```

As simple as that. Just click that button

Comment (Divy123) [reply examples/lib/defaultHtmlSequencerUi.js]: Let me first try if   its working

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlSequencerUi.js]: Btw, How many `.item`s are on the page?

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlSequencerUi.js]: Can there be only a single `.item` or more than 1? What about the insert step dropdown. jQuery will search for everything globally.

Comment (Divy123) [reply examples/lib/defaultHtmlSequencerUi.js]: Well just one!!

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlSequencerUi.js]: Are you sure? I am talking about the whole DOM.

Comment (HarshKhandeparkar) [reply examples/lib/defaultHtmlSequencerUi.js]: What if there are more than a single selectize selectors?

**Comment by Divy123** [comment]
Thanks @HarshKhandeparkar. Made the change!!

**Comment by HarshKhandeparkar** [comment]
What if there are more than a single selectize selectors? Will there be more than a single `.item`?

**Comment by Divy123** [comment]
It works successfully! I checked it locally.

**Comment by Divy123** [comment]
@HarshKhandeparkar nice catch!! Just looking into it. Will make changes if required!!

**Comment by Divy123** [comment]
@HarshKhandeparkar I think there is some error in that part which you said. Morning is my exam. I will see after 12 pm.

**Comment by HarshKhandeparkar** [comment]
Np, exam is your priority.

**Comment by Divy123** [comment]
@jywarren please have a  look!!
@HarshKhandeparkar resolved that part !!

**Comment by jywarren** [comment]
Awesome!

**Pull request merged**